gemfileparser/__init__.py

```diff
@@ -1,6 +1,22 @@
 #!/usr/bin/env python
-
-'''Python library to parse Ruby's Gemfiles and gemspec files.'''
+#
+# Copyright 2015 Balasankar C <balasankarc@autistici.org>
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# .
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# .
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Python library to parse Ruby's Gemfiles and gemspec files.
+"""
```

Comment:
Please leave the formatting with black to a separate PR entirely to avoid any confusion

Reply:
Sure :)